services/galley/src/Galley/Intra/Client.hs

```diff
@@ -0,0 +1,55 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Galley.Intra.Client
+    ( lookupClients
```

Comment:
Any particular reason for moving this function to a separate module, rather than re-using `Intra/User.hs` ? This is also a lookup relating to users, so it would make sense to move it there.

Reply:
It was inspired by the structure in Brig/Data/ which I assume reflects the structure in Cassandra. If we want to fully manage this data in Brig, it would make sense to keep the layout as is (there will be write functions in this module as well).

Moving it to Users would be simpler for this PR ofc.